### PR TITLE
Adding media folder

### PR DESCRIPTION
**Pull request by bruskiza**

### CODE REVIEW

**Comment by bruskiza** [comment]
@smn, @hodgestar @miltontony: ready for review.

File: sideloader/postinstall.sh [hunk 1]
```diff
@@ -8,5 +8,9 @@ npm install -g coffee-script
 
 npm install -g less
 
+if [ ! -d /var/praekelt/casepro/media ]; then
+  mkdir -p /var/praekelt/casepro/media
+fi
```
Comment (smn) [inline sideloader/postinstall.sh]: `-p` means you can run it multiple times and doesn't recreate what exists, the `if` statement in this case is not needed.

**Comment by bruskiza** [comment]
@smn thanks... removed.

**Comment by smn** [comment]
:+1:

**Pull request merged**